.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-  - 2.3.3
+  - 2.4.0
```

Comment:
Elastic Beanstalk doesn't support this ruby version. I think that we must wait a little bit more to update it.